bazel/foreign_cc/BUILD

```diff
@@ -197,109 +197,6 @@ envoy_cmake_external(
     }),
 )
 
-envoy_cmake_external(
-    name = "llvm",
```

Comment:
There's a dependency on the `llvm` target here: https://github.com/maistra/envoy/blob/44abadf7a67d8f0c83676108ce8c84471b635cf7/bazel/foreign_cc/BUILD#L245

Reply:
@dmitri-d yeah, I saw that. Apparently it's not used too. I was able to build (locally) both envoy and proxy without llvm.

Reply:
how weird!